### DISCUSSION TITLE
Make it jump to daily challenge after clicking the "song imported" notification if the beatmap is downloaded from the daily challenge page

### DISCUSSION THREAD

**Discussion by MeerukYang** [ppy/osu#29303] [Ideas] [answered]
Clicking the "song imported" notification would exit daily challenge and jump to the solo playlist in current version, even if the beatmap was downloaded from the daily challenge page. (see in attached video)

https://github.com/user-attachments/assets/8a110876-e80c-4aed-9e8b-38de7dd3a31c

(I was about to submit this as a bug but nvm....)
I think if the song was downloaded from daily challenge page and then imported, it is better to head to daily challenge page since the player's intention is more likely to play the song as daily challenge.
Maybe making the "daily challenge imported" notification differ from other song's import would be a good choice.

**Comment by peppy** [comment]
I'm going to take a quick look into this one.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/29316
